configure

```diff
@@ -1989,6 +1993,25 @@ Available commands are:
       message.index("\n") != nil
     end
   end
+
+  # Returns true if MacPorts is installed in its default location.
+  def macports?
+    File.exists? '/opt/local/bin/port'
```

Comment:
@jc00ke The reason is that MacPorts is intended to be installed under /opt/local, while it's possible that there could be some other command named "port" in the PATH. Looking for the canonical location of the MacPorts command seems safe enough; simply assuming that a successful `which port` means MacPorts is actually installed might lead to odd errors later.